pkgs/development/libraries/libvisual/default.nix

```
@@ -1,51 +1,35 @@
 { lib
 , stdenv
-, fetchurl
+, fetchFromGitHub
 , fetchpatch
 , SDL
 , autoreconfHook
+, autoconf-archive
 , glib
 , pkg-config
 }:
 
 stdenv.mkDerivation rec {
   pname = "libvisual";
-  version = "0.4.1";
+  version = "0.4.2";
 
-  src = fetchurl {
-    url = "mirror://sourceforge/libvisual/${pname}-${version}.tar.gz";
-    hash = "sha256-qhKHdBf3bTZC2fTHIzAjgNgzF1Y51jpVZB0Bkopd230=";
+  src = fetchFromGitHub {
+    owner = "Libvisual";
+    repo = pname;
+    rev = "${pname}-${version}";
+    hash = "sha256-bDnpQODXB2Z6hezVoh7c6cklp6qpyDzVBAnwZD8Gros=";
```

Fetchers should not be bound to pname.

```suggestion
  src = fetchFromGitHub {
    owner = "Libvisual";
    repo = "libvisual";
    rev = "libvisual-${version}";
    hash = "sha256-bDnpQODXB2Z6hezVoh7c6cklp6qpyDzVBAnwZD8Gros=";
```